debian/control

```diff
@@ -735,6 +736,39 @@ Description: RADOS distributed object store client C++ library (development file
  This package contains development files needed for building C++ applications that
  link against librados.
 
+Package: libcephsqlite
+Architecture: linux-any
+Section: libs
+Depends: librados2 (= ${binary:Version}),
+         ${misc:Depends},
+         ${shlibs:Depends},
```

Comment:
i'd suggest drop this line. as `${shlibs:Depends}` substitution variable should be able to take care of this. to be specific, we do call `dh_shlibdeps` in `debian/rules`. and `dh_shlibdeps` should be able to replace `${shlibs:Depends}` with the dependent libraries.

see also https://www.debian.org/doc/debian-policy/ch-sharedlibs.html#s-sharedlibs-depends